### ISSUE TITLE
Web - When Tabbing with Screen Reader JAWS Starts reading at the Top of the Page Again

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#110]
For some reason when I tab on the Steps pages and others my screen reader JAWS starts reading from the top of the page instead of the next items in the tab order — going to open another issue for this problem because it's confusing.

It happens on the main page and others.

On the Step 1 main page after tabbing from the Menu button JAWS reads all the step number some other text I'm not sure where it is, and the all the show and hide content text. I would expect JAWS only to read the bookmark button with one tab, not everything else it is reading. As part of the bookmark icon/link JAWS is reading information about the which bookmark it is followed by "Comply Chain United States Department of Labor" and the rest of the text in the main content section of the page. Not what should be happening. I assume it's happening on other screens too.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

As seen in the screenshot attached, there looks to be a TITLE for the Bookmark button that is getting read by JAWS my screen reader, which most of it does not need to be there. I'm looking at [Step 1: Engage Stakeholders and Partners](https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/3) according to the page but Step 3 for the URL.

Then when I open the first show/hide item of "Learning Objectives", JAWS reads the Step numbers, then the bookmark button information, followed by the Heading, and finally a bunch of text from the banner section before saying "Learning Objectives" is expanded.

I'm thinking either the aria-live region content is not getting cleared or too much is being placed in there, or there is some type of screen reader buffer issue going on.

![ILAB Comply Chain - web - why is there extra text on bookmark button](https://user-images.githubusercontent.com/10423487/91460610-019b2e00-e856-11ea-90a3-4e95620bd202.png)

**Comment by harpreetsdev** [comment]
@jfc3-dol 

We fixed this issue for a) VO to exclude announcing Step 1 - Step10 unnecessarily and b) The Bookmark title has been shortened to only call out the bookmark title and not USDOL seal. 

Although there are still some issues with the Bookmark button.

I'll wait for you to test this issue and provide feedback.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

Testing in IE11 and Chrome with JAWS it seemed to be working for me, so I'm going to close the issue. If other problems arise, we can reopen this issue or create a new one.
